labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -65,8 +118,12 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+    if (seq.length == 0)
+      return false
+    else if (seq.length == 1)
+      return true
+    else
+      return (seq(0) < seq(1)) && checkValuesIncrease(seq.tail)
```

Comment:
This is a solid recursive definition. I wonder if we could have used pattern matching.